OFEP-ofo-flagd-client-support.md

```diff
@@ -0,0 +1,36 @@
+## flagd client support driven by OpenFeature Operator
+
+## State: DRAFTING
+
+Client side applications (e.g. web/mobile) could benefit from access to feature flags. This OFEP outlines the feasibility and benefits of extending [OpenFeature Operator (OFO)](https://github.com/open-feature/open-feature-operator) to manage the deployment of flagd with access from external traffic (e.g. client side applications).
+
+## Background
+
+OFO already manages the deployment of flagd in server side contexts, handling the networking necessary for server side applications to communicate with flagd. This greatly simplifies the steps necessary for administrators to introduce flagd into their cluster, reducing the barrier of entry. Obtaining these benefits for the deployment of externally accessible flagd is the critical motivator for extending OFO.
```

Comment:
> handling the networking necessary for server side applications to communicate with flags

What does OFO do with networking in the K8s cluster for flagD?

>This greatly simplifies the steps necessary for administrators to introduce flagd into their cluster, reducing the barrier of entry. Obtaining these benefits for the deployment of externally accessible flagd is the critical motivator for extending OFO. 

I don't understand this, it sounds like the sidecar deployment pattern makes this a compelling solution and why we should extend it to web clients. Is there another way of phrasing it perhaps?

Reply:
> What does OFO do with networking in the K8s cluster for flagD?

Good point, it doesn't do anything it's just an inherent value of containers within a pod. I will amend this.

> I don't understand this, it sounds like the sidecar deployment pattern makes this a compelling solution and why we should extend it to web clients. Is there another way of phrasing it perhaps?

I have rephrased, please let me know if it's still unclear.